CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to the "Azure Api Management VS Code" extension will be docu
 
```

Comment:
Do we want to follow structure as we have with CLI, so we should at least name the current version

1.0.0b5
+++++
* Add: Support yaml file for `az apic api register` command
* Update: Update some command names, parameter names as well as related descriptions for better understanding. Please leverage `-h` option or refer Azure CLI reference doc to see full list of commands and parameters.

Reply:
Updated, please see whether the new structure is what you expected: https://github.com/microsoft/vscode-apimanagement/blob/04b45b96c29be9734555d9db8d39960026c1e264/CHANGELOG.md